docs/organizer-guide/meetups/faq-meetups.rst

```diff
@@ -0,0 +1,34 @@
+Meetups FAQ
+===========
+
+The following list of questions and answers cover some issues that you might
+encounter when you want to `start a meetup <http://www.writethedocs.org/organizer-guide/meetups/starting/>`_.
```

Comment:
The main meetup page also has this format for links, I'm not sure this is critical at this point. Maybe we should do a general pass at these someday?